docs/docsite/rst/community/development_process.rst

```diff
@@ -23,6 +23,62 @@ Ansible accepts code via **pull requests** ("PRs" for short). GitHub provides a
 
 Because Ansible receives many pull requests, we use an automated process to help us through the process of reviewing and merging pull requests. That process is managed by **Ansibullbot**.
 
+Backport Pull Request Process
+-----------------------------
+
+After the pull request submitted to Ansible for the ``devel`` branch is
+accepted and merged, the following instructions will help you create a
+pull request to backport the change to a previous stable branch.
+
+.. note::
+
+    These instructions assume that ``stable-2.5`` is the targeted release
+    branch for the backport.
+
+.. note::
+
+    These instructions assume that ``https://github.com/ansible/ansible.git``
+    is configured as a ``git remote`` named ``upstream``. If you do not use
+    a ``git remote`` named ``upstream`` adjust the instructions accordingly.
+
+#. Prepare your devel, stable, and feature branches:
+
+   ::
+
+       git checkout devel
+       git --fetch upstream
+       git merge upstream/devel
+       git checkout stable-2.5
+       git pull
```

Comment:
this should be `--rebase` to ensure local changes are applied
this also assumes stable-2.5 was already preexisting on the local repo, i would instead use this step and leave the other as an 'option' if you already have stable-2.5

`git checkout upstream/stable-2.5 -b stable-2.5`

Reply:
It doesn't necessarily assume `stable-2.5` was already pre-exisiting.   The first time you try to checkout a branch, if it's not local, it's pulled from one of the remotes.

In which case, you should never need `--rebase`, since that branch is from `upstream`.

But yes, there are a lot of assumptions when trying to talk git workflow.

Reply:
I've had issues with the branch autoselection, but probably cause i have multiple remotes with the branch,  in any case you should specify which one.

Reply:
if checking out new, there is no need for pull, if the branch existed locally, doing pull w/o a --rebase can create issues (user had local commits)

Reply:
In the end you can reduce most of these lines to 2 steps
```
       git --fetch upstream
       git checkout upstream/stable-2.5 -b cherry-pick/2.5/[PR_NUMBER_FROM_DEVEL]
```